examples/basic.py

```diff
@@ -18,10 +19,8 @@ def __str__(self):
         return self.name
 
 
-async def run():
-    await Tortoise.init(db_url="sqlite://:memory:", modules={"models": ["__main__"]})
-    await Tortoise.generate_schemas()
-
+@init_memory_sqlite
```

Comment:
May be we should make it configurable? To let it pass another `modules` value

Because if you allow only `__main__` I am not sure it would be useful enough to make it part of contrib

Reply:
@abondar I made a refactor to support `models` argument.